content/en/continuous_integration/setup_tests/javascript.md

```diff
@@ -70,56 +59,69 @@ module.exports = {
 
     {{< code-block lang="javascript" filename="testEnvironment.js" >}}
 require('dd-trace').init({
-  service: 'ui-tests' // The name of the Test Service that will appear in the CI Tests tab.
+  service: 'my-ui-app',  // Name of the service or library under test
+  flushInterval: 300000  // To guarantee test span delivery
 })
+
 // jest-environment-jsdom is an option too
 module.exports = require('jest-environment-node')
 {{< /code-block >}}
 
-**Note**: The default configuration should work for most cases, but depending on the volume and speed of your tests, the tracer or the Agent might drop some of the spans. Alleviate this by increasing the `flushInterval` (a value in milliseconds) when initializing the tracer:
+Run your tests as you normally do, specifying the environment where test are being run (e.g. `local` when running tests on a developer workstation, or `ci` when running them on a CI provider) in the `DD_ENV` environment variable. For example:
 
-{{< code-block lang="javascript" >}}
-require('dd-trace').init({
-  flushInterval: 300000
-})
+{{< code-block lang="bash" >}}
+DD_ENV=ci npm test
 {{< /code-block >}}
 
 ### Mocha instrumentation
 
 Add `--require dd-trace/init` to however you normally run your `mocha` tests, for example in your `package.json`:
 
-{{< code-block lang="javascript" >}}
-// package.json
+{{< code-block lang="javascript" filename="package.json" >}}
 'scripts': {
-  'test': 'mocha --require dd-trace/init'
+  'test': 'DD_SERVICE=my-ui-app mocha --require dd-trace/init'
 },
 {{< /code-block >}}
 
-## Disabling instrumentation in local development
```

Comment:
we might want to keep this section. If the instrumentation is added without taking this into account, running local tests will fail, which is annoying (it has been when dogfooding)

Reply:
nvm, it does not fail. It logs an error, but only if you've added the debug flag.